tests/testthat/test-graphics.R

```diff
@@ -189,7 +189,7 @@ test_that("gg_tsdisplay() plots", {
     list(x = "index", y = "value")
   )
 
-  p <- p + ggplot2::labs(x = "x", y = "y", title = "title")
+  p[[1]] <- p[[1]] + ggplot2::labs(x = "x", y = "y", title = "title")
```

Comment:
I don't think this is very related to ggplot2 changes, but I got a message here saying that there is an S3 dispatch conflict between `+.gg` and `+.gg_tsensemble`. This changes I think preserves the spirit of the test, but I don't know if it merits further investigation.

Reply:
Fixed in ef4b3d548bb119540bb412dd2a68969c9db77ac9